content/en/docs/concepts/services-networking/topology-aware-routing.md

```diff
@@ -157,6 +185,17 @@ zone.
   either not picking up on this event, or newly added pods starting in a
   different zone.
 
+
+## Custom Heuristics
+
+Kubernetes is deployed in many different ways, there is no way that a single
+heuristic for allocating endpoints to zones will work for every use case. A
```

Comment:
```suggestion
heuristic for allocating endpoints to zones will work for every use cases. A
```

Reply:
I don't think this is right, at least it doesn't sound quite right to me. I may be missing something though.